### ISSUE TITLE
Relative Links to directories cause fatal error

### ISSUE THREAD

**Issue by ryan-s** [kovetskiy/mark#114]
Having a line like: 
```[`./integration`](integration/):` Integration tests``` 
that is a relative link to a directory in a repository causes the following fatal error:
```
2021-09-09 13:23:33.173 TRACE found a relative link: full=integration filename=integration hash=
2021-09-09 13:23:33.173 FATAL unable to resolve relative links
                              └─ resolve link: "integration"
                                 └─ read file: integration
                                    └─ read integration: is a directory
```

Expected: Relative links that are directories are ignored
Actual: Relative links that are directories cause a fatal error, preventing file from syncing to confluence.

**Comment by ryan-s** [comment]
Looking at this the error seems to be coming from Karma.  This made me unsure if this was expected behavior or behavior that was inherited from the dependency.

**Comment by kovetskiy** [comment]
Hi, you are right, currently it's expecting that the link is possible only
to a file, it even reads the content and makes sure it's a file. It does so
to make links on *documents* in Confluence. How do you expect it to work
this link on a directory in Confluence? It should not crash with a fatal
error tho.

чт, 9 сент. 2021 г. в 17:18, Ryan Schaffer ***@***.***>:

**Comment by ryan-s** [comment]
I can think of 2 potential behaviors:
1. I think the base case would be just to ignore the link and only upload the text to confluence
2. Alternatively, and this would probably be more of a feature-request, you could give the user the ability to pass a hosted repository url as an argument and it would generate a link to the directory url inside of the hosted repository.

**Comment by kovetskiy** [comment]
Should be fixed in version 6.3, let me know if it doesn't work
